configure.py

```diff
@@ -393,6 +393,7 @@ def find_headers(repodir, excluded_dirs):
     'test/boost/virtual_reader_test',
     'test/boost/bptree_test',
     'test/boost/double_decker_test',
+    'test/boost/stall_free_test',
```

Comment:
Missing corresponding change to test.py.

Reply:
Change test.py for what? Do I need to list stall_free_test to test.py? I do not see individual test in test.py is mentioned.

Reply:
Right, that was the old way. The new test.py does not require listing.

Reply:
Cool.